install/linux/usr/share/odemis/hwtest/xt-only.odm.yaml

```diff
@@ -1,52 +1,57 @@
 SEM: {
-  class: Microscope,
-  role: sem,
-  children: ["EBeam Scanner", "EBeam Focus", "EBeam Stage",
-             "EBeam Detector"],
+    class: Microscope,
+    role: sem,
+    children: ["EBeam Scanner", "EBeam Focus", "EBeam Stage",
+               "EBeam Detector", "Chamber"],
 }
 
 # Connects to TFS Microscopes, the Support PC should be running the
 # Delmic XTAdapter. Update the address to match that of the Support PC.
 "SEM EXT": {
-  class: xt_client.SEM,
-  role: null,
-  init: {
-    address: "PYRO:Microscope@localhost:4242",
-  },
-  children: {
-    scanner: "EBeam Scanner",
-    fib-scanner: "FIB Scanner",
```

Comment:
Please leave this line as comment too. "fib-scanner" is the important part actually.

Reply:
done